package.json

```diff
@@ -3,15 +3,21 @@
   "version": "0.33.0",
   "description": "Stream-based MQTT broker",
   "main": "aedes.js",
+  "types": "types/index.d.ts",
   "scripts": {
     "lint": "standard",
+    "tslint": "tslint types/**/*.d.ts",
+    "typescript-compile-test": "tsc -p test/typescript/tsconfig.json",
+    "typescript-compile-execute": "node test/typescript/*.js",
+    "typescript-test": "npm run typescript-compile-test && npm run typescript-compile-execute",
     "test": "tape test/*.js test/*/*.js | faucet",
-    "ci": "npm run lint; npm run coverage",
+    "ci": "npm run lint;  npm run typescript-test; npm run coverage",
```

Comment:
Can you use `&&` instead of `;`?

Reply:
Sure, I was just sticking to what was already in place :)